frontend/src/components/GeocacheList/GeocacheList.tsx

```diff
@@ -8,6 +8,9 @@ interface Props {
     geocaches: Array<Geocache>
 }
 
+//const ICON_DIMENSIONS = [36 / 1.5, 27 / 1.5];
+const ICON_DIMENSIONS = [18 / 1, 13.5 / 1];
```

Comment:
No need for division, if it's a division by 1. Also the comment can be removed. Otherwise looks good

Reply:
I got back to original [36 / 1.5, 27 / 1.5];. Icons are now slightly larger, divide by 2 is also good  [36 / 2, 27 / 2].